### PR TITLE
feat: add graceful shutdown to pool_noodle, cleanup logs lines

### PR DESCRIPTION
**Pull request by sprutton1**

### CODE REVIEW

**Comment by sprutton1** [comment]
bors try

File: lib/deadpool-cyclone/examples/deadpool-cyclone-ping-pool/main.rs [hunk 1]
```diff
@@ -33,7 +34,9 @@ async fn main() -> Result<(), Box<(dyn std::error::Error + 'static)>> {
     };
 
     let spec = spec()?;
-    let mut pool: PoolNoodle<LocalUdsInstance, _> = PoolNoodle::new(10, spec.clone());
+    let (shutdown_broadcast_tx, _) = broadcast::channel(16);
```
Comment (paulocsanz) [inline lib/deadpool-cyclone/examples/deadpool-cyclone-ping-pool/main.rs:37]: `tokio_util::sync::CancellationToken` may be a better fir for the shutdown channel, but it's not a blocker

Comment (sprutton1) [reply lib/deadpool-cyclone/examples/deadpool-cyclone-ping-pool/main.rs:37]: I used broadcast as that's what Veritech already uses for its graceful shutdown bits, so it was easiest to just leverage what was already there

**Pull request merged**